techlibs/xilinx/xc7_xcu_brams.txt

```diff
@@ -85,6 +88,9 @@ match $__XILINX_RAMB36_SDP
 endmatch
 
 match $__XILINX_RAMB18_SDP
+  attribute ram_style=block ram_block=1
+  attribute !ram_style
```

Comment:
Can you double check this? My reading is that this condition is:
If ram_style == block OR ram_block == 1
AND ram_style attribute is not present # <<< Do the tests show this to be a conflict?
AND logic_block attribute is not present